community/contributing/docs/github-changes.md

```diff
@@ -9,10 +9,10 @@ sidebar:
 
 # Open a pull request on GitHub
 
-1. Navigate to the [Spinnaker documentation repository](https://github.com/spinnaker/spinnaker.github.io). Note that the top navigation of Spinnaker.io corresponds to folder names. Use them as a reference to help you find the file you're looking for.
-2. On the page where you see the issue, select the pencil icon at the top right. You can also scroll to the bottom of the page and select **Edit this page**.
-3. Make your changes in the GitHub markdown editor.
-4. Below the editor, fill in the **Propose file change** form.
+1. Click the **Suggest an Edit** link in the upper right hand corner of the Spinnaker documentation page. This takes you to the file in GitHub.
```

Comment:
```suggestion
1. Click the **Suggest an Edit** link on the documentation page you want to update. This takes you to the file in GitHub.
```

Reply:
The reason I suggest this change is that directional descriptions like "upper right" are considered less accessible. Also, if we move the link this'll be broken. But unless we change the text of the link, this will still be findable.